tools/rosbag-inspector/CMakeLists.txt

```diff
@@ -48,6 +48,8 @@ if(BUILD_GRAPHICAL_EXAMPLES)
         FOLDER Tools
     )
 
+    using_easyloggingpp( rs-rosbag-inspector SHARED )
```

Comment:
Please explain

Reply:
I added a debug log to a macro in a header that's used in this project, on `common/os.cpp`. So it needed to be declared to use EasyLogging, as other projects do.